test/e2e/broker_defaults_webhook_test.go

```diff
@@ -65,6 +65,10 @@ func TestBrokerNamespaceDefaulting(t *testing.T) {
 		}
 
 		defaults["namespaceDefaults"][c.Namespace] = map[string]interface{}{
+			"apiVersion":  "v1",
+			"kind":        "ConfigMap",
+			"name":        "config-br-default-channel",
+			"namespace":   "knative-eventing",
```

Comment:
I am guessing this is the real fix to #4721, right? If so, should we merge #4720 with this one?

Reply:
I discovered this while working on #4720 , however, IMHO they are different issues and hence I made this into a separate PR.

Fortunately/unfortunately the test will still pass with all the right assertions without this PR as the broker object is created with the correct annotation and spec attributes. However, without this change
- the logs are noisy
- it might cause folks more confusion and might act as a red herring

Reply:
These are valuable points. Can you give an example of logs before and after your change in this PR for a record please?

I do prefer small changes too, just thought they together fix the issue.

Reply:
Yes, for sure. Added them in the PR description.